### PR TITLE
Created and Added App Assets and Icons

### PR DESCRIPTION
**Pull request by xinyao-song**
Screenshot 2024-11-20 at 1 30 20 PM

### CODE REVIEW

**Review by zhenbi93 — APPROVED** [review]
LGTM

**Comment by AllenHsm** [comment]
LGTM

File: GOLDTracker/src/__tests__/LoginScreen.test.js
Comment (AllenHsm) [inline GOLDTracker/src/__tests__/LoginScreen.test.js:1]: Needs to be removed?

**Review by AllenHsm — APPROVED** [review]
Good

**Pull request merged**